plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -646,6 +651,9 @@ protected String getNetworkDirectDevice() {
         return _networkDirectDevice;
     }
 
+    protected String getDefaultTungstenScriptsDir() {
+        return "scripts/vm/network/tungsten";
```

Comment:
What do you think of making it a constant instead of a method?
A method would be interesting if necessary outside of this class (as it is protected instead of private I assume that this is the idea).

I was not able to find occurrences outside of this class but I might be missing something here.

Reply:
Done. I added a function call that was missing before

Reply:
Got it, thanks for the update @huyle-uni!